Project.toml

```diff
@@ -4,6 +4,12 @@ authors = ["Charles Kawczynski <kawczynski.charles@gmail.com>"]
 version = "0.4.0"
 
 [deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+EnsembleKalmanProcesses = "aa8a2aa5-91d8-4396-bcef-d4f2ec43552d"
```

Comment:
Can we somehow not add this dependence?

Reply:
This will be here until `ParameterDistributions` are removed from `EnsembleKalmanProcesses` would you allow merging before this is done?

Even once removed, this package will depend on `ParameterDistributions.jl` or whatever we call it.

Reply:
Why would this package depend on `ParameterDistributions.jl`? Will Thermodynamics need functionality that its offering?